.buildkite/pipeline.yml

```diff
@@ -168,6 +168,13 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":computer: CRM rcemipii in a box with 1M"
```

Comment:
Would it make sense to add it to reproducibility tests and get a warning every time results change? I think this is one of the better tests for microphysics we have in Atmos now

Reply:
And is it ok to remove some aquaplanet 1M / 2M tests after this is merged? If so I can do it in a separate PR.